### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by nathancoleman**
### Description
Add generated changelog entry for v1.19.1

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [x] appropriate backport labels added
* [x] not a security concern

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+* ui: Pin and namespace sub-module dependencies related to the Consul UI [[GH-21378](https://github.com/hashicorp/consul/issues/21378)]
+
+FEATURES:
+
+* ui: Added a banner to let users link their clusters to HCP [[GH-20275](https://github.com/hashicorp/consul/issues/20275)]
+
+IMPROVEMENTS:
+
+* mesh: update supported envoy version 1.29.4
+* mesh: update supported envoy version 1.29.5 in addition to 1.28.4, 1.27.6. [[GH-21277](https://github.com/hashicorp/consul/issues/21277)]
```
Comment (sarahalsmiller) [inline CHANGELOG.md:26]:
```suggestion
* mesh: update supported envoy version 1.29.5 in addition to 1.29.4, 1.28.4, 1.27.6. [[GH-21277](https://github.com/hashicorp/consul/issues/21277)]
```

**Review by sarahalsmiller — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@nathancoleman, a backport is missing for this PR [21537] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Pull request merged**